CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## unreleased
 * Remove `ENABLE_BITCODE` from framework target build settings
   * _The App Store no longer accepts bitcode submissions from Xcode 14_
+* BraintreePayPalNativeCheckout (BETA)
+   * Post Notification to PayPalCheckout SDK to update Analytics Config
```

Comment:
I don't think we need a changelog since this change is not merchant facing

Reply:
Changelog is removed